dora/core/common/src/main/java/alluxio/Constants.java

```diff
@@ -175,6 +175,7 @@ public final class Constants {
   public static final String MODE_BITS_READ_EXECUTE = "r-x";
   public static final String MODE_BITS_READ_WRITE = "rw-";
   public static final String MODE_BITS_ALL = "rwx";
+  public static final String FILE_SEPARATOR = "/";
```

Comment:
unused? also wouldn't this be a system constant?

Reply:
plugged into places need to use it, must have been lost during rebasing etc.

Reply:
i would recommend adding this as a constant within the membership package rather than a global constant
- i doubt this will be reused for most instances of the correct scenario of "separating file paths". so it introduces an inconsistency in where this is being used

rename to MEMBERSHIP_PATH_SEPARATOR
- it's not actually separating files. it's usage is for separating path parts in membership names

check through all instances of `"/"` in the membership code to ensure it is used correctly
- i found at least one place that it should be used that currently isn't

Reply:
moved to membershipmanager to be visible to all membership package